### PR TITLE
feat: add bot code base (!!!)

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

This PR is a the bot's code base as a package. 🎉 

xref: https://github.com/regro/cf-scripts/issues/2894

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin relint

**Comment by beckermr** [comment]
I made a comment from the bot account in a show of good faith that we actually do control this account.

**Comment by beckermr** [comment]
@conda-forge/help-python @conda-forge/staged-recipes this is a `noarch: python` recipe that is ready for review!

File: recipes/conda-forge-tick/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+test:
+  imports:
+    - conda_forge_tick
+  commands:
+    - python -c "import conda_forge_tick; assert conda_forge_tick.__version__ == '{{ version }}'"
+
+about:
+  home: https://github.com/regro/cf-scripts
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (synapticarbors) [inline recipes/conda-forge-tick/meta.yaml:84]: Summary looks to be left over from the simplejson example recipe

Comment (beckermr) [reply recipes/conda-forge-tick/meta.yaml:84]: arg every time!

Comment (beckermr) [inline recipes/conda-forge-tick/meta.yaml:84]:
```suggestion
  summary: 'Flagship repo for cf-regro-autotick-bot'
```

Comment (beckermr) [reply recipes/conda-forge-tick/meta.yaml]: fixed!

**Pull request merged**